pkgs/applications/science/physics/shtns/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, stdenv, fetchgit, fftw } :
+
+stdenv.mkDerivation rec {
+  pname = "shtns";
+  version = "v3.5.1";
+
+  src = fetchgit {
+    url = "https://bitbucket.org/nschaeff/shtns";
+    rev = "${version}";
```

```suggestion
    rev = "v${version}";
```
btw, you can set `rev = version;` directly when you don't need to include a versioning prefix.